andela/vof-deployment-scripts
# #163384175 Include new login keys as environment vars in image creation

#### What does this PR do?
Adds new environment variables in terraform scripts and in VOF image creation.
#### Description of Task to be completed?
- update `terraform/vof/compute.tf` with the required variables
- create new terraform variables `terraform/vof/variables.tf` to hold the keys
- update the start script to export the newly environment variables

#### How should this be manually tested?
- `clone` this repo
- `git checkout ft-include-new-login-keys-163384175`
- deploy this to local folk or sandbox
- `SSH` into the sandbox via `jumpbox` to check if they have been set

#### Any background context you want to provide?
There are keys that are required if user login is supposed to work properly. These keys include
```
STAGING_ANDELA_MICRO_PUBLIC_KEY
STAGING_LEARNER_MICRO_PUBLIC_KEY
PRODUCTION_ANDELA_MICRO_PUBLIC_KEY
PRODUCTION_LEARNER_MICRO_PUBLIC_KEY

```
#### What are the relevant pivotal tracker stories?
[163384175](https://www.pivotaltracker.com/story/show/163384175)
#### Screenshots (if appropriate)

#### Questions:
